sonar-application/src/main/assembly/conf/sonar.properties

```diff
@@ -217,9 +217,60 @@
 #--------------------------------------------------------------------------------------------------
 # LOGGING
 
-# Level of logs. Supported values are INFO(default), DEBUG and TRACE (DEBUG + SQL + ES requests)
+# SonarQube produces logs in 4 logs files located in the same directory (see property sonar.path.logs below),
+# one per JVM: App (main process), Web Server (aka. Web), Compute Engine (aka. CE) and Elastic Search (aka. ES)
+# All 4 files have logs in the same format:
+#           1           2    3           4                       5                                                   6
+# |-----------------| |---| |-|--------------------||------------------------------| |------------------------------------------------------------------------------------------------------------------------------|
+# 2016.11.16 16:47:00 INFO  ce[AVht0dNXFcyiYejytc3m][o.s.s.c.t.CeWorkerCallableImpl] Executed task | project=org.sonarqube:example-java-maven | type=REPORT | id=AVht0dNXFcyiYejytc3m | submitter=admin | time=1699ms
+#
+# 1: timestamp. Format is YYYY.MM.DD HH:MM:SS
+#    YYYY: year on 4 digits
+#    MM: month on 2 digits
+#    DD: day on 2 digits
+#    HH: hour of day on 2 digits in 24 hours format
+#    MM: minutes on 2 digits
+#    SS: seconds on 2 digits
+# 2: log level. Possible values are (in order of descending criticality): ERROR, WARN, INFO, DEBUG and TRACE
+# 3: process identifier. Possible values are: app (main process), web (Web Server), ce (Compute Engine) and es (Elastic Search)
+# 4: thread identifier. Can be empty.
```

Comment:
I'll go for "SQ Thread identifier" or something of the like as it makes sense to reference the Thread concept. As so, fields go from larger to most precise in a sense that any one can understand : process -> thread -> logger